templates/web-application/README.md

```diff
@@ -0,0 +1,64 @@
+# {{PACKAGE-NAME}}
+
+> Your web-application information go here
+
+## Scripts
+
+### dev
+
+Start a development server at the address [http://localhost:8080](http://localhost:8080).
+
+```bash
+pnpm dev
+```
+
+### build
+
+Build {{PACKAGE-NAME}} as production ready and output the result in `dist`;
+
+```bash
+pnpm build
+```
+
+### test
+
+Run the tests on the sections of {{PACKAGE-NAME}}.
+
+```bash
+pnpm test
+```
+
+### serve-build
+
+Run a local server on the content of the `dist` folder at the address [http://localhost:8080](http://localhost:8080).
+
+```bash
+pnpm serve-build
+```
+
+### reset
+
+Delete all build results and `pnpm` installed packages in this project.
+
+```bash
+pnpm reset
+```
+
+Run linting on {{PACKAGE-NAME}} files.
+
+### lint
+
+```bash
+pnpm lint
+```
+
+### analyze
+
+Run Webpack with bundle analyzer on the result of a production-ready build.
+
+```bash
+pnpm analyze
+```
+
+---
+Build with [wl-foundry-cli](https://github.com/workleap/wl-foundry-cli) from [web-application template](https://github.com/workleap/wl-foundry-cli/tree/main/templates/web-application) 🚀
```

Comment:
New file, based on cammand already available in https://github.com/patricklafrance/pnpm-ts5-swc-esm-jest/tree/master